pep-0683.rst

```diff
@@ -0,0 +1,342 @@
+PEP: 683
+Title: Immortal Objects, Using a Fixed Refcount
+Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eelizondo@fb.com>
+PEP-Delegate: <TBD>
```

Comment:
For rendering it would be better to leave the `<TBD>` fields out whilst still undecided